management/appointments/appointmentsController.js

```diff
@@ -91,6 +93,7 @@ define('appointmentsController', [], function() {
 
 		$scope.selectAppointment = function(appointment) {
 			$scope.appointment = appointment;
+			document.body.scrollTop = document.documentElement.scrollTop = 0;
```

Comment:
Just ease-of-use things that will make the screen jump back to the top when a volunteer clicks on a client or they reschedule an appointment on the appointment management page.